easybuild/easyblocks/m/mvapich2.py

```diff
@@ -51,8 +51,6 @@ def extra_options():
         """Define custom easyconfig parameters specific to MVAPICH2."""
         extra_vars = {
             'withchkpt': [False, "Enable checkpointing support (required BLCR)", CUSTOM],
-            'withmpe': [False, "Build MPE routines", CUSTOM],
-            'withhwloc': [False, "Enable support for using hwloc support for process binding", CUSTOM],
```

Comment:
these shouldn't be removed

although we can update the easyconfigs we ship ourselves accordingly, we should forget that there are other easyconfigs out there we do not control...